internal/jsonwall/log.go

```diff
@@ -0,0 +1,53 @@
+package jsonwall
+
+import (
+	"fmt"
+	"sync"
+)
+
+// Avoid importing the log type information unnecessarily.  There's a small cost
+// associated with using an interface rather than the type.  Depending on how
+// often the logger is plugged in, it would be worth using the type instead.
+type log_Logger interface {
+	Output(calldepth int, s string) error
+}
+
+var globalLoggerLock sync.Mutex
+var globalLogger log_Logger
+var globalDebug bool
+
+// Specify the *log.Logger object where log messages should be sent to.
```

Comment:
Should this line start with the function name? (may apply to more locations in his file.)

Reply:
Definitely should. Per above, let's please fix it all at once.